homeassistant/components/zwave_js/update.py

```diff
@@ -103,7 +103,7 @@ def async_add_firmware_update_entity(node: ZwaveNode) -> None:
     )
 
 
-class ZWaveNodeFirmwareUpdate(UpdateEntity):
+class ZWaveNodeFirmwareUpdate(UpdateEntity, RestoreEntity):
```

Comment:
UpdateEntity already inherits RestoreEntity.

https://github.com/home-assistant/core/blob/94e192db12723e2fd7d9ecbb5a9807ff80ea15f1/homeassistant/components/update/__init__.py#L192

Reply:
ah, thanks, clearly didn't review closely enough. OK I will need to find a way to reproduce locally